C-Plus-Plus/cp/Dividing_tasks.cpp

```diff
@@ -0,0 +1,37 @@
+/*
+The Problem states that Given array arr[n] of integer where each element arr[i] represent number of task given to each student.Your task is to find if it possible to redistribute the task where i'th student assign i number of task. Example is it possible to assign 2 task to 2nd Student. 
+*/
```

Comment:
Rewrite it like that

```
/*
The Problem states that Given array arr[n] of integer where each element arr[i] represent 
number of task given to each student.Your task is to find if it possible to redistribute the 
task where i'th student assign i number of task. Example is it possible to assign 2 task to 2nd Student. 
*/
```

Reply:
Resolve this issue

Reply:
"The problem state that given n number of students in the class and for i'th student given arr[i] represent number of task assigned to that student. Teacher who assigned task to them is getting confuse with the numbers. So she decided to redistribute task in such a way that ,i'th student will get i number of task ( example, 2nd student will get 2 tasks).
Your task is to find if it is possible to redistribute the task. Help the teacher!"

@ankitaggarwal23 Is this sound good mentor or any changes

Reply:
No just change it as mentioned rest looks good